flask_redis_sentinel.py

```diff
@@ -91,8 +96,19 @@ def slave_for(self, service_name, **kwargs):
 
         conn = sentinel.slave_for(service_name, redis_class=self.client_class, **kwargs)
         self.local.slave_connections[service_name] = conn
+        self._connections.put(conn)
         return conn
 
+    def disconnect(self):
```

Comment:
It seems to me that disconnect is not thread safe, could you please document it?